dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/utils/ConfigValidationUtils.java

```diff
@@ -602,6 +603,11 @@ public static void validateProtocolConfig(ProtocolConfig config) {
                 checkMultiExtension(config.getScopeModel(), Codec2.class, CODEC_KEY, config.getCodec());
                 checkMultiExtension(
                         config.getScopeModel(), Serialization.class, SERIALIZATION_KEY, config.getSerialization());
+                checkMultiExtension(
+                        config.getScopeModel(),
+                        Serialization.class,
+                        PREFER_SERIALIZATION_KEY,
+                        config.getPreferSerialization());
```

Comment:
The value of `prefer.serialization` should be splitted first cause it can be `fastjson2,hessian2`.

Reply:
[ConfigValidationUtils.java#L695](https://github.com/apache/dubbo/blob/3.2/dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/utils/ConfigValidationUtils.java#L695) value has been splitted